modules/cassandra/src/main/java/org/apache/ignite/cache/store/cassandra/CassandraCacheStore.java

```diff
@@ -370,12 +385,12 @@ public CassandraCacheStore(DataSource dataSrc, KeyValuePersistenceSettings setti
      */
     private CassandraSession getCassandraSession() {
         if (storeSes == null || storeSes.transaction() == null)
-            return dataSrc.session(log != null ? log : new NullLogger());
+            return dataSrc.session(log);
```

Comment:
It will produce NullPointerException if Ignite was started with Basic configuration without Spring support added (see the same comment above)